src/hooks/useList.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback, useState } from '@redneckz/uni-jsx/lib/hooks';
+
+export function useList<T>(defaultValue: T[] = []): [
+  T[],
+  {
+    setValue: (_: T[]) => void;
+    push: (item: T) => void;
+    pop: () => void;
+    remove: (item: T) => void;
+    clear: () => void;
+  },
+] {
+  const [value, setValue] = useState<T[]>(defaultValue);
```

Comment:
Может быть
```suggestion
  const [list, setList] = useState<T[]>(defaultValue);
```

И таким же точно образом назвать и выходные параметры?

Reply:
++